frontend/src/views/web/challenge/phases.html

```diff
@@ -3,9 +3,13 @@
         <div class="row">
             <div class="col s11 m11">
                 <div class="row">
-                    <div class="col s12">
+                    <div class="col s4">
```

Comment:
I am not sure how this will look like? can you please add a GIF/Video showing the timer?

Reply:
I have intentionally left this code if in future we might need it. Moreover, the video is at the channel on the slack.

Reply:
> I have intentionally left this code if in future we might need it. 

Let's not keep the commented code here. We can always implement it. It's quite simple and straightforward. 

> Moreover, the video is at the channel on the slack.

The PRs are public and we should make sure that all the PR related details are there in the PR description. :)

Reply:
> Let's not keep the commented code here. We can always implement it. It's quite simple and straightforward.
Sure! I'll remove it and update the PR.

> The PRs are public and we should make sure that all the PR related details are there in the PR description. :)
I will add this too!!

Reply:
This is also not done. Please let me know when are done with all the above cahnges.